src/app/components/layouts/tag/terra-tag.component.ts

```diff
@@ -45,22 +101,22 @@ export class TerraTagComponent
      */
     private getBgColor():string
     {
-         if(!isNullOrUndefined(this.inputColor))
-         {
-             return this.inputColor;
-         }
-         return null;
+        if(!isNullOrUndefined(this.inputColor))
```

Comment:
if macht keinen Sinn... kannst auch direkt `return this.inputColor;` machen, wenn es null ist, dann returned es null...

Reply:
was hat eig. BackgroundColor mit color zu tun?

Reply:
Da hat nur der Autoformatter zugeschlagen, deswegen keine Ahnung. Würde das nur ungern jetzt noch anpassen (testing und so)